src/web-components.js

```diff
@@ -48,6 +48,28 @@ export function isShadowDomSupported() {
 }
 
 /**
+ * Returns `true` if Shadow CSS encapsulation is supported.
+ * @return {boolean}
+ */
+export function isShadowCssSupported() {
+  if (!isShadowDomSupported()) {
+    return false;
+  }
+
+  //TODO: Find a better way to test CSS encapsulation
```

Comment:
Done. 

If I pass in the shadow root element that is created in `shadow-embed.js` we get `[object DocumentFragment]`, since We polyfill the element ourselves.